.docker/wp-config.php

```diff
@@ -118,6 +118,8 @@
 define( 'EMAIL_DEVELOPER_NOTIFICATIONS', 'developers@example.test' );
 define( 'EMAIL_CENTRAL_SUPPORT',         'support@wordcamp.test' );
 
+define( 'ORGANIZER_SURVEY_ACCESS_TOKEN_KEY', 'wcorg_debrief_survey_email' );
```

Comment:
```suggestion
define( 'ORGANIZER_SURVEY_ACCESS_TOKEN_KEY', 'put your unique phrase here' );
```

This might make it more obvious that the value is essentially a password, so it should be a long randomly generated  value, just like the key/salt values above.